helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroCompatibilityHelper.java

```diff
@@ -27,20 +39,14 @@
 import org.apache.avro.io.JsonEncoder;
 import org.apache.avro.specific.SpecificRecord;
 
-import com.linkedin.avroutil1.compatibility.avro14.Avro14Adapter;
-import com.linkedin.avroutil1.compatibility.avro15.Avro15Adapter;
-import com.linkedin.avroutil1.compatibility.avro16.Avro16Adapter;
-import com.linkedin.avroutil1.compatibility.avro17.Avro17Adapter;
-import com.linkedin.avroutil1.compatibility.avro18.Avro18Adapter;
-import com.linkedin.avroutil1.compatibility.avro19.Avro19Adapter;
-
 
 /**
  * a Utility class for performing various avro-related operations under a wide range of avro versions at runtime.
  */
 public class AvroCompatibilityHelper {
   private static final AvroVersion DETECTED_VERSION;
   private static final AvroAdapter ADAPTER;
+  private static final FieldBuilder FIELD_BUILDER;
```

Comment:
why is this required? why not add a method - cloneField() into the adapter interface that returns a field builder?

Reply:
I thought for avro-utils, we should not return the "internal" class but return a type in AVRO libraries, like Schema, or Decoder etc.